.github/workflows/deploy.yml

```diff
@@ -0,0 +1,33 @@
+name: "Deploy to Netlify"
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+    name: Publish release to Netlify
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v1
+      - name: Use Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - name: Build stuff
+        run: |
+          npm install
+          npm run redoc
+          npm run build
+      - name: push builds to `deployment` branch
+        uses: actions/checkout@v2
+        with:
+          ref: deployment
```

Comment:
are you sure you're pushing to `deployment` here? o.o cuz it just looks like a checkout